e2e-tests/tests/infra/spawnStudio.ts

```diff
@@ -23,7 +23,7 @@ export default async function spawnStudio(
   const port = await getPort(testInfo);
   const child = spawn(
     "npx",
-    ["studio", "--port", port.toString(), "--root", rootDir],
+    ["studio", "--port", port.toString(), "--root", rootDir, "--", "--noGenerateTestData"],
```

Comment:
right now studio should never try to generate test data on startup, mostly because we want it to run in CBD (i.e. the code editor's live preview) but also because we're still trying to limit the amount of PagesJS specific logic